src/cljs/rems/application.cljs

```diff
@@ -448,13 +448,11 @@
   [{:keys [element-id attributes application group? can-remove? accepted-licenses?]}]
   (let [application-id (:application/id application)
         user-id (or (:eppn attributes) (:userid attributes))
-        sanitized-user-id (-> (or user-id (:email attributes) "") ;; use email for invited members
-                              str/lower-case
-                              (str/replace #"[^a-z]" ""))
+        user-id-or-email (or user-id (:email attributes) "") ; use email for invited members
```

Comment:
I've been reading this for some time and testing using the UI, but I can't figure out what this change does and why it's needed. The commit message says "fix all forms being opened with a single button". Which forms, which button? I can't find a button on Create Application page that would open several forms at once. Can you clarify?

Reply:
The old version removed everything except letters from the user-id, so that completely valid, distinct user ids and emails got the same id. When collapsibles have same id, bootstraps operates on them all.

Test by adding perftester1 and perftester2, for example.

Reply:
Ok, thanks. I was mislead by how the commit message was formulated (strictly speaking, "all forms" are not necessarily opened, several forms can in some cases be).